### ISSUE TITLE
Handling textures inside executables (.DOL, .REL)

### ISSUE THREAD

**Issue by Neui** [Venomalia/DolphinTextureExtraction-tool#31]
Executables can have textures (and other files) in them, but they are directly accessed by code and thus harder to "find".
For example, Pokémon XD Gale of Darkness has some GTX textures in its .DOL file that it "loads" during startup.

Since we can't really rely on trying to search for possible textures by searching for magic values (since formats like GTX doesn't have one), maybe we would have an separate table that for a given executable (identified by file size and some hash) it would show where a texture is located in and what format (via filename?), so that those can also be extracted.

**Comment by Venomalia** [comment]
Yes I have thought about it, but mostly it's just the startup logos.
I don't think it's worth the effort as they can be dumped very quickly with dolphin.
